test/test.js

```diff
@@ -0,0 +1,38 @@
+const puppeteer = require('puppeteer');
+const tap = require('tap');
+const pa11y = require('pa11y');
+
+async function test(server){
```

Comment:
Nice asinc/await tests like this.
Need to be separated (integration/Aria) when expanded.

Reply:
Could be, as well. But I meant separation in different files/folders.